mrmustard/math/__init__.py

```diff
@@ -13,23 +13,25 @@
 # limitations under the License.
 
 r"""
-The `math` module contains low-level functions for performing mathematical operations. It is recommeded
-that users access the backends using the an instance of the Math class rather than the backends themselves.
+The :code:``math`` module contains low-level functions for performing mathematical operations.
+
+It is recommeded that users access the backends using the an instance of the :class:`Math` class rather than the backends themselves.
```

Comment:
```suggestion
It is recommended that users access the backends using the an instance of the :class:`Math` class rather than the backends themselves.
```